src/app/src/components/NavBar.js

```diff
@@ -88,9 +88,7 @@ function ExitButton({ variant }) {
         </Button>
     ) : (
         <Button
-            onClick={() => {
-                navigate('/submissions');
-            }}
+            onClick={() => navigate(-1)}
             rightIcon={<Icon as={ArrowLeftIcon} />}
         >
             Save and back
```

Comment:
Possibly out of scope: Should this read as just `Back` if the user has neither `canReview` or `canWrite`?

Reply:
[5e42173](https://github.com/azavea/iow-boundary-tool/pull/156/commits/5e42173b230d2921c49f4ed388c1650794b5619c)

Reply:
This is a creative solution to a difficult problem, however, I agree we'll need to harden it in the future. Reaching into the RTK query cache is depending on implementation details of an external library.